### PR TITLE
WaitForPodsReady: Add documentation for the backoffMaxSeconds

### PR DESCRIPTION
**Pull request by tenzen-y**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
We don't have any documentation about https://github.com/kubernetes-sigs/kueue/pull/2264.

#### Which issue(s) this PR fixes:

Part-of #2216

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md [hunk 1]
```diff
@@ -77,9 +78,15 @@ When the `blockAdmission` is set to `true`, admitted Workload with not ready pod
 {{% alert title="Warning" color="warning" %}}
 _Available in Kueue v0.6.0 and later_
 {{% /alert %}}
+{{% alert title="Note" color="primary" %}}
+_The `backoffBaseSeconds` and `backodMaxSeconds` are available in Kueue v0.7.0 and later_
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/setup_sequential_admission.md:82]:
```suggestion
The `backoffBaseSeconds` and `backodMaxSeconds` are available in Kueue v0.7.0 and later
```

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md [hunk 2]
```diff
@@ -77,9 +78,15 @@ When the `blockAdmission` is set to `true`, admitted Workload with not ready pod
...
+{{% /alert %}}
 
 The `requeuingStrategy` (`waitForPodsReady.requeuingStrategy`) contains optional parameters: 
-`timestamp` (`waitForPodsReady.requeuingStrategy.timestamp`) and `backoffLimitCount` (`waitForPodsReady.requeuingStrategy.backoffLimitCount`).
+- `timestamp` (`waitForPodsReady.requeuingStrategy.timestamp`)
+- `backoffLimitCount` (`waitForPodsReady.requeuingStrategy.backoffLimitCount`)
+- `backoffBaseSeconds` (`waitForPodsReady.requeuingStrategy.backoffBaseSeconds`)
+- `backoffMaxSeconds` (`waitForPodsReady.requeuingStrategy.backoffMaxSeconds`)
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/setup_sequential_admission.md:89]:
```suggestion
- `timestamp`
- `backoffLimitCount`
- `backoffBaseSeconds`
- `backoffMaxSeconds`
```

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md [hunk 3]
```diff
@@ -94,13 +101,13 @@ If you don't specify any value for `backoffLimitCount`,
...
 
-{{% alert title="Note" color="primary" %}}
-_The `backoffBaseSeconds` is available in Kueue v0.7.0 and later_
-{{% /alert %}}
 The time to re-queue a workload after each consecutive timeout is increased
 exponentially, with the exponent of 2. The first delay is determined by the
 `backoffBaseSeconds` parameter (defaulting to 60). So, after the consecutive timeouts
-the evicted workload is re-queued after approximately `60, 120, 240, ...` seconds.
+the evicted workload is re-queued after approximately `60, 120, 240, ..., 3600, 3600 ..., 3600` seconds 
+if you don't specify the maximum backoff time by the `backoffMaxSeconds` (defaulting to 3600).
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/setup_sequential_admission.md:108]:
```markdown
`backoffBaseSeconds` parameter (defaulting to 60). You can configure the maximum
backoff time by setting `backoffMaxSeconds` (defaulting to 3600). Using the defaults, the
evicted workload is re-queued after approximately `60, 120, 240, ..., 3600, ..., 3600` seconds.
```

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md [hunk 4]
```diff
@@ -94,13 +101,13 @@ If you don't specify any value for `backoffLimitCount`,
...
+Even if the backoff time reaches the `backoffMaxSeconds`, Kueue will keep to re-queue an evicted Workload with the `backoffMaxSeconds`
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/setup_sequential_admission.md:109]:
```suggestion
Even if the backoff time reaches the `backoffMaxSeconds`, Kueue will continue to re-queue an evicted Workload with the `backoffMaxSeconds`
```

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md (hunk 3 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/setup_sequential_admission.md:108]: Thank you for great suggestion!

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md (hunk 1 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/setup_sequential_admission.md]: Done.

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md (hunk 2 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/setup_sequential_admission.md]: Done.

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md (hunk 3 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/setup_sequential_admission.md]: Done.

File: site/content/en/docs/tasks/manage/setup_sequential_admission.md (hunk 4 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/setup_sequential_admission.md]: Done.

**Comment by tenzen-y** [comment]
@alculquicondor I addressed all comments. PTAL, thanks!

**Review by alculquicondor — COMMENTED** [review]
/lgtm

**Pull request merged**